appserver/monitoring-console/core/src/main/java/fish/payara/monitoring/alert/Circumstance.java

```diff
@@ -137,4 +141,30 @@ public String toString() {
         }
         return str.toString();
     }
+
+    public JsonValue toJSON() {
```

Comment:
I know this class already exists, but a description of what it's used for wouldn't go amiss - circumstance isn't an intuitive name to me in the context of alerts (in comparison to something like _event_).

Reply:
Added javadoc.